### PR TITLE
Log fields for client Connection ID and HTTP/2 Stream ID.

### PR DESCRIPTION
**Pull request by ywkaras**
%**<**ccid**>** and %**<**ctid**>**, respectively.

### CODE REVIEW

**Comment by scw00** [comment]
[approve ci]

**Comment by ywkaras** [comment]
I'm unable to reproduce the autest failure.

File: doc/admin-guide/logging/formatting.en.rst [hunk 1]
```diff
@@ -179,19 +179,28 @@ Connections and Transactions
...
+sca   Proxy          Number of attempts within the current transaction by |TS|
+                     in connecting to the origin server.
+sstc  Proxy          Number of transactions between the |TS| proxy and the origin
+                     server from a single session. Any value greater than zero
+                     indicates connection reuse.
+ccid  Client Request Client Connection ID, a non-negative number for a connection,
+                     which is different for all currently-active connections to
+                     clients.
+ctid  Client Request Client Transaction ID, a non-negative number for a transaction,
+                     which is different for all currenlty-active transactions on the
```
Comment (SolidWallOfCode) [inline doc/admin-guide/logging/formatting.en.rst]: typo - `currently`

**Comment by SolidWallOfCode** [comment]
[approve ci autest]

**Comment by SolidWallOfCode** [comment]
Based on the output it looks like the failure was a timing failure where the log file didn't get flushed. We need to do something about that.

**Comment by SolidWallOfCode** [comment]
[approve ci]

**Comment by SolidWallOfCode** [comment]
Wow, you actually crashed TS. This might be a real problem.

**Comment by SolidWallOfCode** [comment]
[ci approve autest]

**Comment by SolidWallOfCode** [comment]
[ci approve autest]

**Comment by SolidWallOfCode** [comment]
[approve ci autest]

**Comment by SolidWallOfCode** [comment]
I was able to replicate the TS crashes locally but it's a shut down effect, not during the basic testing.  As best I can tell it's crashing in `ProcessManager::stop` at
```
ink_release_assert(running == 1);
```

**Comment by zwoop** [comment]
This would require #2508 as well, which is a significant feature addition (low risk, but feature).

**Pull request merged**